### PR TITLE
Typo fix

### PR DESCRIPTION
**Pull request by yuki462-b**
## What this PR does / why we need it

Just fixed some typos I found. Some of the corrections were made in anticipation, so if I am wrong, please point it out.

Verified that the code passes format and testing.

```bash
$ tox -e fmt,lint
...
All checks passed!
  fmt: OK (3.75=setup[1.80]+cmd[1.95] seconds)
  lint: OK (0.48=setup[0.44]+cmd[0.04] seconds)
  congratulations :) (4.32 seconds)

$ tox -e py312
Required test coverage of 85.0% reached. Total coverage: 85.71%

================================================================== 857 passed in 100.22s (0:01:40) ===================================================================
  py312: OK (103.62=setup[2.00]+cmd[101.62] seconds)
  congratulations :) (103.70 seconds)
```

## What gif most accurately describes how I feel towards this PR?

Example of a gif

### CODE REVIEW

**Review by HonakerM — APPROVED** [review]
LGTM! Thank you for fixing all of these typos (I know a fair amount of them were mine 😅).

Probably want a review from @gabe-l-hart before merging incase there are backwards compatibility issues at play

**Review by gabe-l-hart — APPROVED** [review]
Ship it! Thanks for all these fixes

**Pull request merged**
